tools/quadrados/src/init.cpp

```diff
@@ -0,0 +1,134 @@
+#include <filesystem>
+#include <fstream>
+#include <iostream>
+
+#include "tools.hpp"
+
+namespace fs = std::filesystem;
+
+static void printHelp()
+{
+    std::cerr << "Usage: quadrados init <PROJECT NAME> [OPTIONS]" << std::endl;
+    std::cerr << "Options:" << std::endl;
+    std::cerr << "  -p <PATH>   Specifies the path of project directory." << std::endl;
```

Comment:
I would add an option here for specifying the template the user wants to use for the project. By the default, it would be the `simple` template (which is the only one we have right now). You could also maybe have a way to list the existing templates?